Containerfile

```diff
@@ -0,0 +1,52 @@
+FROM quay.io/devto/ruby:2.7.1
+
+USER root
+
+RUN curl -sL https://dl.yarnpkg.com/rpm/yarn.repo -o /etc/yum.repos.d/yarn.repo && \
+    dnf install -y bash git ImageMagick iproute less libcurl libcurl-devel \
```

Comment:
You and me both...

Reply:
TLDR; there might be a light at the end of the tunnel to get rid of ImageMagick: `libvips`

While going through Rails 6 changelog I discovered the [image_processing](https://github.com/janko/image_processing) (default for ActiveStorage in Rails 6) library which is a layer on top of IM and [libvips](https://github.com/libvips/libvips) which is a faster alternative to ImageMagick: <https://github.com/libvips/libvips/wiki/Speed-and-memory-use> which I totally forgot about but knew from when I read https://images.guide back then when Addy Osmani published it.

`image_processing` in turn uses [ruby-vips](https://github.com/libvips/ruby-vips) which is also used by [carrierwave-vips](https://github.com/libvips/ruby-vips) which could be used by us instead of ImageMagick (we currently use IM only to strip EXIFs before sending the image up to S3)